platform.md

```diff
@@ -931,6 +934,7 @@ type = "<process type>"
 command = "<command>"
 args = ["<arguments>"]
 direct = false
+working-directory = "<working directory>"
```

Comment:
Has this flag been implemented yet?

I noticed it changed after approvals were already made in the [RFC](https://github.com/buildpacks/rfcs/pull/144), and I think we should consider something more standard (especially next to other short names like `args`). E.g.,  `working-dir`

Reply:
It has been implemented, but it is easy to change. We could make it `working-dir` if that's preferred.